charts/spire/values.yaml

```diff
@@ -102,9 +97,16 @@ server:
     upstreamAuthority:
       disk:
         enabled: false
-        certFilePath: /conf/server/tls.crt
-        keyFilePath: /conf/server/tls.key
-        bundleFilePath: ""
+        secret:
+          # -- If disabled requires you to create a secret with the given keys yourself.
+          create: true
+          # -- If secret creation is disabled will look for a secret with this name.
+          name: "spiffe-upstream-ca"
```

Comment:
Better change the ordering here. Have the data first (since that is mandatory with the default).